### DISCUSSION TITLE
Trouble displaying cover art and chapter names

### DISCUSSION THREAD

**Discussion by wag357** [seanap/Plex-Audiobook-Guide#37] [Q&A] [answered]
Thanks for this guide and for your project to help with AudioBooks in Plex!

I have read through your instructions and have been trying for days to resolve 2 issues.
1 is I can't get the chapter names to show, it just has the title of the book for each chapter
mp3tag seems to be pulling data from Audible as the rest of the fields populate

The second issue is that I can not get the cover art to show up in the Plex clients nor in Bookcamp for Android
the cover files ARE in the folders and I've even tried converting the JPGs to PNGs to see if it was a format issue.

I attached some screenshot which may help. What the flip am I doing wrong? Thanks for any advice!

![ChapterNames](https://user-images.githubusercontent.com/1805482/135960214-68bb9772-c62f-4ff9-bee9-237fd4b78a9b.png)
![AudiblePullConfig](https://user-images.githubusercontent.com/1805482/135960911-a97c3891-cad6-42b8-a715-013f4dbea918.png)
![SuggestedFileNameConfig](https://user-images.githubusercontent.com/1805482/135960912-b9d239f6-716a-435a-a8ac-43f62aed2164.png)

**Comment by seanap** [comment, marked as answer]
Hey, sorry your having issues, it doesn't looks like you are doing anything wrong.  There is no db of chapters and chapters aren't available on audible.com for me to scrape.  So unfortunately we can't set the chapter names automatically.  There is another project in development that is using the audible api to set the chapter names but it's not finished yet.

So the covers aren't showing up at all? What version PMS are you running? There was a major bug in 1.24.3 that was fixed in 1.24.4

**Reply by wag357** [reply]
Thanks for your reply!
Ok, I'm glad to know I didn't miss something to get the chapters to show.
The covers aren't showing no matter what I do and I've tried every which way I could think to try.
I am running 1.24.3.5033 and the update checker says I'm on the latest version.  I'll check their site in the morning and see if I can get 1.24.4 manually.
Thanks for the suggestion.  I'll reply again after I've updated.  Fingers crossed xD

**Reply by wag357** [reply]
I decided sleep was for the weak :) and went to the Plex site and found 1.24
I installed it and my beautiful covers are showing up!  Thank you!

**Comment by seanap** [comment]
Glad you got it working! For future reference incase anyone else comes across this thread, the linuxserver.io docker does pull 1.24.4 when used with the "latest" tag (eg. Linuxserver/plex:latest ) even though there isn't a specific 1.24.4 tag for the new release yet.
